components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,57 @@
-import React from "react";
-import { Text, View } from "react-native";
+import { useTheme } from "@react-navigation/native";
+import React, { useState } from "react";
+import { Text, View, Button } from "react-native";
 
-export default function StateMutipleChoiceQuizPage() {
+type Question = {
+  questionText: string;
+  option: string[];
+  correctAnswer: string;
+};
+const quizQuestions: Question[] = [
+  {
+    questionText: "What is Karnataka's capital?",
+    option: ["Bangalore", "Chennai", "Hyderabad", "Raipur"],
+    correctAnswer: "Bangalore",
+  },
+];
+
+const StateMultipleChoiceQuizApp: React.FC = () => {
+  const [currentQuestionIndex, setCurrentQuestion] = useState<number>(0);
+  const [score, setScore] = useState<number>(0);
+  const [quizFinished, setQuizFinished] = useState<boolean>(false);
+
+  const currentQuestion = quizQuestions[currentQuestionIndex];
+
+  const handleAnswer = (selectedOption: string) => {
+    if (selectedOption === currentQuestion.correctAnswer) {
+      setScore(score + 1);
+    }
+    if (currentQuestionIndex < quizQuestions.length - 1) {
+      setCurrentQuestion(currentQuestionIndex + 1);
+    } else {
+      setQuizFinished(true);
+    }
+  };
   return (
     <View>
-      <Text>Pick the correct answer</Text>
+      {quizFinished ? (
+        <View>
+          <Text>Quiz Finished</Text>
+          <Text>Your Score: {score}</Text>
+        </View>
+      ) : (
+        <View>
+          <Text>{currentQuestion.questionText}</Text>
+          {currentQuestion.option.map((option) => (
+            <Button
+              key={option}
+              title={option}
+              onPress={() => handleAnswer(option)}
+            ></Button>
+          ))}
+        </View>
+      )}
     </View>
   );
-}
+};
+export default StateMultipleChoiceQuizApp;
```

Comment:
Add a line break before the `export` statement.

Reply:
I have removed the export statement at the bottom to the top, where the function is being defined, in the recent commit. Check once